services/federator/test/integration/Test/Federator/Monitor.hs

```diff
@@ -0,0 +1,26 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2021 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Test.Federator.Monitor (spec) where
+
+import Imports
+import Test.Federator.Util
+import Test.Hspec
+
+spec :: TestEnv -> Spec
+spec _env =
+  describe "Monitor" $ pure ()
```

Comment:
Is this an unfinished test?

Reply:
It was a stub for the integration test I was planning to write. I removed it for now. See 52a357c54377fc039ded6868d4330fce17851e58.